require relative paths

I'm using atom with [hydrogen plugin](https://github.com/willwhitney/hydrogen), basically is jupyter for the text editor and I have installed your package successfully and everything is working just fine.

![screen shot 2015-05-27 at 15 17 20](https://cloud.githubusercontent.com/assets/6867996/7838386/7df1a8be-0484-11e5-8cfc-58788bd5d415.png)

The problem now is with require for relative paths, which I guess ipython won't know the PATH I'm on and wont pass it to node, does any one know a workaround for this?

```js
var test = require('./test');
```

Error: Cannot find module './test'
at Function.Module._resolveFilename (module.js:336:15)
at Function.Module._load (module.js:278:25)
at Module.require (module.js:365:17)
at require (module.js:384:17)
at Object.Contextify.sandbox.run (/Users/rbarros/package/node_modules/contextify/lib/contextify.js:12:24)
at Context.rawRun (/Users/rbarros/package/build/context.js:105:23)
at Context.rawEvaluate (/Users/rbarros/package/build/context.js:117:23)
at Context.execute (/Users/rbarros/package/build/context.js:194:21)
at Kernel.executeRequest (/Users/rbarros/package/build/kernel.js:153:16)
at Kernel.onShell (/Users/rbarros/package/build/kernel.js:90:14) Runtime error
Cannot find module './test'

--- a/lib/context.js
+++ b/lib/context.js
@@ -18,6 +18,13 @@ export default class Context {
         if (fs.existsSync(full)) {
           return require(full)
         }
+
+        const start = name.substring(0, 2)
+
+        if (start === './' || start === '..') {
+          name = path.resolve(process.env.PWD, name)
+        }
+
         return require(name)
       },
       Buffer,
@@ -284,4 +291,3 @@ function format(val) {
     'text/plain': toSafeString(val)
   }
 }
-

--- a/lib/context.js
+++ b/lib/context.js
@@ -22,7 +22,7 @@ export default class Context {
         const start = name.substring(0, 2)
 
         if (start === './' || start === '..') {
-          name = path.resolve(process.env.PWD, name)
+          name = path.resolve(name)
         }
 
         return require(name)

